transformer_engine/jax/csrc/modules.cpp

```diff
@@ -877,10 +868,9 @@ void RMSNormForwardFP8(cudaStream_t stream, void **buffers, const char *opaque,
     auto *scale_inv = reinterpret_cast<float *>(buffers[4]);
     auto *output = buffers[5];
     auto *rsigma = buffers[6];
-    auto *amax_out = buffers[7];
+    // auto *amax_out = buffers[7];  // Bound to amax in primitive
```

Comment:
@denera Is there a reason some of these are commented out and not deleted?

Reply:
These were raising unused variable warnings at compile even despite a C assert on `amax == amax_out`, but I thought outright deleting them would be confusing to anyone else who isn’t aware that we bind these to each other in the accompanying JAX primitive implementation. So I left them commented to maintain a record of why XLA is giving us pointers to buffers that appear unused when in fact they are not. 

@zlsh80826 recently told me that NVTE_CHECK() instead of the C assert eliminates the compile warning, so I think we can uncomment it and go back to doing the check on pointer equivalency like before.